### ISSUE TITLE
Copying one or more blocks from site A to site B

### ISSUE THREAD

**Issue by jeffreyschmitz87** [spicywebau/craft-neo#897]
### Bug Description

A customer of ours reports that pasting a block on site B from a copied block on site A produces an error.

They use this method so that a (large) content block can be copied to another site (language) to adjust it there. This saves a lot of time because the block does not have to be made from scratch.

The propagation method for this section is set to 'none' and the latest version of Craft (4.x) and Neo (4.x) is used.

### Steps to reproduce

1) Press the 'Copy' method on a block from site A
2) Switch to site B
3) Edit an entry and use the 'Paste' method
4) An error is thrown

```php
{
    "name": "Unsupported site",
    "message": "Attempting to save an element in an unsupported site.",
    "code": 0,
    "error": "Attempting to save an element in an unsupported site.",
    ....
}
```

### Neo version

4.2.2

### Craft CMS version

4.10.0

### What is the affected Neo field's propagation method?

None

### Does this issue involve templating, and if so, is eager-loading used?

This is not a templating issue

**Comment by ttempleton** [comment]
Thanks for reporting that - fixed for the next 4.x release.

**Comment by ttempleton** [comment]
That fix is now included in Neo 4.2.3.
